File: OutliersAPP.API/Models/playlist.cs
using System;
using System.Collections.Generic;
using CloudinaryDotNet.Actions;

namespace OutliersAPP.API.Models
{
    public class playlist
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Description { get; set; }
        public DateTime DateAdded { get; set; }
        public User User { get; set; }
        public int UserId { get; set; }
        public ICollection<video> Videos{ get; set; }
    }
}

File: OutliersAPP.SPA/src/app/_models/playlist.ts
import { Video } from "./video";

export interface Playlist {
  Id:number;
  Name:string;
  dateAdded:Date;
  userPhotoUrl:string;
  videoUrl:string;
  description:string;
  category:string;
  profName: string;
  profCity: string;
  profCountry: string;
  profInfo: string;
  profPhone:string;
  videos?: Video[];

}
